src/main/engine/services/browser.ts

```diff
@@ -62,6 +76,7 @@ export class BrowserService extends Browser {
             });
         }
         this.emit('attached');
+        await playwrightSetCurrentPagePromise;
```

Comment:
Why don't we wait for the playwright's promise before emitting the event?

Reply:
Good point, it shouldn't really matter because reactions on attached event are solely using internal CDP, but let's just move it at the end to avoid confusion since we're anyway awaiting promise.